electron/main.mjs

```diff
@@ -96,6 +96,13 @@ function createWindow(url) {
     // Allow navigation for internal URLs
     return { action: 'allow' };
   });
+
+  win.webContents.on('will-navigate', (event, url) => {
+    if (url.startsWith('file://')) {
+      event.preventDefault();
+      shell.openExternal(url);
```

Comment:
Any reason to not just use the default of the system like the other button? Is it because this catches all navigation events?

Reply:
I tested and the one above doesn't work but this works. I got this from chatgpt.